src/commands/rm.py

```diff
@@ -8,14 +8,21 @@ class Rm:
 
     @staticmethod
     def do(shell: Shell, line: str):
+        parameter, _ = shell.get_arguments(line)
```

Comment:
Replace `_` by `options`, the underscore was meant for cases in which the options are not used (avoid an unused variable)